building/product/unlocking-exercises.md

```diff
@@ -0,0 +1,26 @@
+# Unlocking Exercises
+
+Exercises are unlocked differently based on the type of track:
+
+1. Tracks without Learning Exercises: all Practice Exercises are unlocked from the start.
+2. Tracks with Learning Exercises: complete Learning Exercises to unlock further Learning and Practice Exercises.
+
+## Learning Mode vs Practice Mode
+
+Requiring Learning Exercises to be completed to unlock more exercises is called Learning Mode (which was introduced with Exercism 3.0).
+Learning Mode is the default for any user that joins a track with Learning Exercises, but also for any users from the old site.
+Practice Mode is the opposite, where all exercises are unlocked from the start.
```

Comment:
This only seems to apply to practice exercise, not all exercises, unless that's a bug or I'm missing something.

Reply:
I've just tested this locally, but to me all exercises become unlocked. Not for you?

Reply:
Hmmm, weird! Thanks for reporting